src/libraries/System.Runtime/tests/System.Dynamic.Runtime.Tests/Dynamic.Context/Conformance.dynamic.context.method.regmethod.regclass.cs

```diff
@@ -714,7 +714,7 @@ public static void CalledFrom_UsingExpression()
             {
                 using (StreamWriter sw = new StreamWriter(sm))
                 {
-                    sw.WriteLine(mc.Method_ReturnString('a'));
+                    sw.WriteLine((string)mc.Method_ReturnString('a'));
```

Comment:
This is to comply with 

> error CS9220: (NETCORE_ENGINEERING_TELEMETRY=Build) One or more overloads of method 'WriteLine' having non-array params collection parameter might be applicable only in expanded form which is not supported during dynamic dispatch.

I think this means this PR would be a breaking change, no @stephentoub, @AlekseyTs?

Reply:
I don't understand the error. @AlekseyTs?

Reply:
> I don't understand the error. 

It is not an error, it is a warning. I assume the ```WriteLine``` is invoked dynamically, one of possibly applicable overloads has non-array `params` parameter. C# runtime binder is unable to consume it, hence the warning. This is specified at https://github.com/dotnet/csharplang/blob/main/proposals/params-collections.md#new-rules
> - Otherwise, if any candidate passing the test has non-array params parameter and it could possibly be applicable only in an expanded form, a compile-time warning occurs.

Reply:
The build probably uses warn as error mode.

Reply:
You probably don't want to change the test if its goal is to test dynamic invocation, simply suppress the warning for the call site.

Reply:
Thanks for explaining. One thing I don't understand.

> it could possibly be applicable only in an expanded form

Wouldn’t this case be considered impossible, since the overload with non-array params is `WriteLine(string format, params ReadOnlySpan<object?> arg)` i.e: two arguments, and the dynamic expression uses `WriteLine` with one argument.

Reply:
The expanded form includes the case of no arguments matching the `params` parameter. An empty collection is supposed to be passed then.

Reply:
Ah, right! Thanks!